src/Operation/ListIndexes.php

```diff
@@ -123,7 +123,7 @@ private function createOptions()
      *
      * @param Server $server
      * @return IndexInfoIteratorIterator
-     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     * @throws CommandException for other driver errors (e.g. connection errors)
```

Comment:
These two instances of `@throws` should not have been modified. Did you think to do this because of a phpcs warning that DriverRuntimeException was undefined? IIRC, it does attempt to check class references in doc block comments.

Remember that [ServerException](https://www.php.net/manual/en/class.mongodb-driver-exception-serverexception.php) and its child classes are defined as originating from a server response. Something like a connection error (mentioned on this very line) can still be thrown by [`Server::executeReadCommand()`](https://www.php.net/manual/en/mongodb-driver-server.executereadcommand.php) and `ListIndexes::executeCommand()` will not catch it at all and allow it to propagate up the chain.

See [Errors/Exceptions in `ServerException::executeReadCommand()`](https://www.php.net/manual/en/mongodb-driver-server.executereadcommand.php#refsect1-mongodb-driver-server.executereadcommand-errors) for a list of possible exceptions that the method might throw. Note that it doesn't mention ServerException directly, but that's covered by RuntimeException.

----

On a separate note, if these lines _were_ actually supposed to be modified, we would also want to update the documentation for [Errors/Exceptions in `Collection::listIndexes()`](https://docs.mongodb.com/php-library/master/reference/method/MongoDBCollection-listIndexes/#errors-exceptions), the source of which is in [docs/reference/method/MongoDBCollection-listIndexes.txt](https://github.com/mongodb/mongo-php-library/blob/master/docs/reference/method/MongoDBCollection-listIndexes.txt).

Reply:
Ah I got confused and changed it while I was changing the DriverRuntimeException to CommandException, but now I see I shouldn't have modified it in the @throws. Sorry, it makes sense to me now!